test/unit/atomic/src/AtomicTest.cpp

```diff
@@ -37,9 +37,9 @@ ALPAKA_FN_ACC auto testAtomicAdd(
                 acc,
                 &operand,
                 value);
-    ALPAKA_CHECK(*success, operandOrig == ret);
+    ALPAKA_CHECK(*success, operandOrig == ret)
```

Comment:
please reintroduce the semicolon and enforce it with `do{}while(true)` or `((void) 0)`